### PR TITLE
Adding fieldConfig overrides in table panel

### PR DESCRIPTION
**Pull request by vishnuchalla**
### Description
Adding filterable option in table panel. More context here: https://github.com/grafana/grafonnet/issues/126

### Testing
Tested by building a grafana panel. Could see expected json and filter option in the table panel.

### CODE REVIEW

**Comment by vishnuchalla** [comment]
@K-Phoen @iainlane Requesting a review.

**Comment by Duologic** [comment]
It exists in the schema, I've been looking through docs for relevance of this field, we can enable it here instead: https://github.com/grafana/grafonnet/blob/main/generator/core.libsonnet#L271

File: gen/grafonnet-v10.0.0/clean/panel/table.libsonnet
Comment (Duologic) [inline gen/grafonnet-v10.0.0/clean/panel/table.libsonnet:1]: Note the first line in this file: ```// This file is generated, do not manually edit.```

**Comment by Duologic** [comment]
Fixing this in #131

**Comment by vishnuchalla** [comment]
> Fixing this in #131

I still do not understand the usage of it. It tried both 
* `table.fieldConfig.defaults.withFilterable`
```
RUNTIME ERROR: field does not exist: fieldConfig
```
* `table.standardOptions.withFilterable`
```
RUNTIME ERROR: field does not exist: withFilterable
```

I think the field is not properly exposed yet for table panel atleast.

**Comment by Duologic** [comment]
Did you do a `jb update`?

**Comment by vishnuchalla** [comment]
I did update and retry this.
when I do a `table.fieldConfig.defaults.withFilterable`, I still see this error
```
RUNTIME ERROR: field does not exist: fieldConfig
```
And when I do `table.standardOptions.withFilterable`, the filter seems to be getting applied but in a wrong place i.e `fieldConfig.defaults.filterable` instead of `fieldConfig.defaults.custom.filterable`. So I am unable to see filter options in my dashboard.

**Comment by Duologic** [comment]
Gotcha, so there is a distinct field `fieldConfig.defaults.custom.filterable` that is not in the table schema. Makes sense to investigate that deeper, I find a [reference here](https://github.com/grafana/grafana/blob/ff85ec33c56ffe567e6bde27473d9493eb70c743/packages/grafana-schema/src/common/table.cue#L88) which shows that there are more fields missing. Let's see how we can get those into Grafonnet too.

**Comment by Duologic** [comment]
#134 makes an attempt to fix this again.

**Comment by Duologic** [comment]
#134 got merged

**Pull request closed without merging**